specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -2582,6 +2582,249 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/triggers/{triggerName}/rerunTriggers/{rerunTriggerName}": {
+      "put": {
+        "tags": [
+          "rerunTriggers"
+        ],
+        "operationId": "RerunTriggers_Create",
+        "x-ms-examples": {
+          "RerunTriggers_Create": {
+            "$ref": "./examples/RerunTriggers_Create.json"
+          }
+        },
+        "description": "Creates?a?rerun?trigger.",
```

Comment:
Please fix the descriptions (is there a reason why `?` is used instead of space)